src/content/1.7/themes/getting-started/asset-management/_index.md

```diff
@@ -40,7 +40,9 @@ In order to have the most extensible signatures, these 2 methods take 3 argument
 
 **ID**
 
-This unique identifier needed for each asset. This is useful to either override or unregister something already loaded by the Core or a native module.
+This unique identifier needed for each asset. Reusing the same ID is useful to either override or unregister something already loaded by the Core or a native module.
+
+However, avoid generic names when adding new JS and CSS files to avoid colision with other extensions. Prefixing with your module or theme name is a good start.
```

Comment:
```suggestion
However, avoid generic names when adding new JS and CSS files to avoid collision with other extensions. Prefixing with your module or theme name is a good start.
```

So we want to tell people to avoid doing something like `$this->registerStylesheet('script');` or `this-> registerStylesheet('jqueryplugin');` ?

Reply:
Indeed.

This method has to be called with more than one parameter, but yes, we want generic values to be avoided as several modules could use the same, making the first file added overwritten by the second one.

Reply:
Thanks for the suggestion